Client/mainImage.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+
+
+class MainImage extends React.Component {
+  constructor(props) {
+    super(props)
+
+  }
+
+  render () {
+
+    return (
+
+      <div>
+      <div>
+        <img
+          className="mainImage"
+          src={this.props.image.url}
+         alt={this.props.image.alt}
+         ></img>
+      </div>
+
+      </div>
```

Comment:
I don't think you need the outer div, unless you'll be adding something else later.